civitai-downloader-v2/README.md

```
@@ -0,0 +1,277 @@
+# Check performance report
+report = manager.optimizer.get_performance_report()
+print(f"Network condition: {report['metrics']['network_condition']}")
+print(f"Recommendations: {report['recommendations']}")
+```
+
+### Analytics and Reporting
+
+```python
+from core.analytics import create_complete_analytics_system, quick_analytics_report
+
+# Initialize analytics system
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)